lua/astrocommunity/media/codesnap-nvim/init.lua

```diff
@@ -0,0 +1,24 @@
+local prefix = "<Leader>c"
+return {
+  "mistricky/codesnap.nvim",
+  build = "make",
```

Comment:
```suggestion
  build = "build_generator",
```
We probably shouldnt use a library compiled with someone else…

Reply:
tested and it doesn't work well. I think this is a fine default